### ISSUE TITLE
DOC202 false positive with abstract methods

### ISSUE THREAD

**Issue by MichaReiser** [astral-sh/ruff#12893]
I really like these new rules, they provide a lot of extra guardrails for docstrings!

When trying to start using rule `DOC202` I ran into a bug. When defining an abstract method, the rule raises an error, because nothing is returned (which makes sense, since it's an empty method, the real code is in the subclasses).

As a small example, an error is raised for this method:

```py
    @property
    @abc.abstractmethod
    def realised_prices(self) -> pd.DataFrame:
        """
        The realised prices of the market.

        Returns:
            The realised prices as a DataFrame
        """
```

_Originally posted by @RubenVanEldik in https://github.com/astral-sh/ruff/issues/12434#issuecomment-2288967263_

**Comment by MichaReiser** [comment]
This should be resolved by https://github.com/astral-sh/ruff/pull/12771 We plan to get a release out today or tomorrow

**Comment by RubenVanEldik** [comment]
Sorry, I missed the PR that solved this issue, absolutely amazing work!! ❤️❤️❤️
